recipe/meta.yaml

```diff
@@ -8,18 +8,15 @@ source:
   fn: libkml-{{ version }}.tar.gz
   url: https://github.com/libkml/libkml/archive/{{ version }}.tar.gz
   sha256: 8892439e5570091965aaffe30b08631fdf7ca7f81f6495b4648f0950d7ea7963
-  patches:
+  patches:              # [win]
     # Patches for vc14. If this works we should wrapit in a ifdef and send the patch upstream.
-    - empty_char.patch  # [win and py>35]
-    - uriparse.patch
```

Comment:
The file https://github.com/AnacondaRecipes/libkml-feedstock/blob/851fa541bc5f1565ac18ae1f70ab26a8997e8c72/recipe/uriparse.patch can be removed from the recipe folder too